docs/source/api/v5/deliveryservice_requests_id_status.rst

```diff
@@ -261,7 +261,7 @@ The response is a full representation of the modified :term:`DSR`.
 			"innerHeaderRewrite": null,
 			"ipv6RoutingEnabled": null,
 			"lastHeaderRewrite": null,
-			"lastUpdated": "0001-01-01T00:00:00Z",
+			"lastUpdated": "2022-09-29T20:58:53-6:00",
```

Comment:
Do we not need milliseconds?

Reply:
This is the Http Post Body Data. We cannot add milliseconds in this case. This data is part of requested {} data that is returned to http operations.